### DISCUSSION TITLE
sun-panel辅助工具箱导出的书签文件后缀json，在项目里导入时为什么显示灰色不能导入？

### DISCUSSION THREAD

**Discussion by jikuohong** [hslr-s/sun-panel#115] [Q&A]
导出的备份文件也一样。
![image](https://github.com/hslr-s/sun-panel/assets/34758287/aa265c4e-54dc-4318-ac97-229fb808ec49)

**Comment by jikuohong** [comment]
最后文件发给手机，手机上传的。

**Reply by tiancool** [reply]
点显示选项，全部显示就可以导入了
